src/main/java/org/folio/circulation/resources/DeclareLostResource.java

```diff
@@ -45,6 +48,7 @@ private void declareLost(RoutingContext routingContext) {
         .thenCompose(r -> r.after(loan -> lostItemFeeService
           .chargeLostItemFees(loan, request, context.getUserId()))))
       .thenCompose(r -> r.after(storeLoanAndItem::updateLoanAndItemInStorage))
+      .thenComposeAsync(r -> r.after(eventPublisher::publishDeclaredLostEvent))
```

Comment:
I'm not sure how it is important, but in some circumstances loan can be declared lost and immediately closed

Reply:
Do you mean it can be closed as part of this process or the loan closure can be triggered separately?

Reply:
The loan can be closed as part of the process if no lost fee is charged (see `lostItemFeeService.chargeLostItemFees` above)

Reply:
For our current purpose it is not important - we only care whether the item is lost. But thanks for this information, it may help us in the future.